suplemon/modules/battery.py

```diff
@@ -71,7 +72,8 @@ def battery_status_read(self):
     def battery_status_acpi(self):
         """Get the battery status via acpi."""
         try:
-            raw_str = subprocess.check_output(["acpi"])
+            FNULL = open(os.devnull, "w")
```

Comment:
We never closed this file descriptor =/

We should either use `with` to prevent doing it by hand or add a `close`:

```python
with open(os.devnull, "w") as FNULL:
    raw_str = subprocess.check_output(["acpi"], stdout=FNULL, stderr=FNULL)
```